### DISCUSSION TITLE
Your suggestions for a project related to ImageMagick

### DISCUSSION THREAD

**Discussion by muhammadnaqdi** [ImageMagick/ImageMagick#5150] [General] [answered]
Hello,

I recently graduated from university and was looking for a brand new C++ project to work on as a side project. Then ImageMagick Caught my eye. It's a lovely piece of software. I remember it from my childhood. And I used to like it.

I was thinking about making a modern GUI. But I'm also wide open to your suggestions.

Regards

**Comment by urban-warrior** [comment, marked as answer]
Look for issues labeled `up for grabs`.  Those are issues that all require development.   A modern GUI would be an excellent project.  We have a recent MagickCache project that is a hidden-gem that needs to evolve (e.g. search the repository for similar images).  We still need someone to port the ImageMagick build to `CMake` or `Meson`.  Some users have made initial attempts at a build, but walked away before the build was robust enough for us to fold it into the project.  Also, since you know C++, Magick++ needs to continue to evolve.

Have fun.
